d2game/d2gamescreen/main_menu.go

```diff
@@ -83,6 +83,142 @@ type BuildInfo struct {
+	allRightsReservedLabel
+	singlePlayerLabel
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	battleNetLabel
+	otherMultiplayerLabel
+	exitLabel
+	creditsLabel
+	cinematicsLabel
+
+	// cinematics menu labels
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	viewAllCinematicsLabel
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	epilogueLabel
+	selectCinematicLabel
+
+	// multiplayer menu labels
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	openBattleNetLabeL
+	tcpIPGameLabel
+	tcpIPOptionsLabel
+	tcpIPHostGameLabel
+	tcpIPJoinGameLabel
+	tcpIPEnterHostIPLabel
+	tcpIPYourIPLabel
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	tipHostLabel
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	tipJoinLabel
+	ipNotFoundLabel
+
+	// select hero class menu labels
+	charNameLabel
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	hardCoreLabel
+	selectHeroClassLabel
+	amazonDescr
+	necromancerDescr
+	barbarianDescr
+	sorceressDescr
+	paladinDescr
+
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	notUsed1 // this position isn't used here.
+
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	hellLabel
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	nightmareLabel
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	normalLabel
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	selectDifficultyLabel
+
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	notUsed2
+
+	delCharConfLabel
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	openLabel
+	//nolint:deadcode,varcheck,unused // will be used, or is need
+	notUsed3
+	yesLabel
+	noLabel
+)
+
+func baseLabelNumbers(idx int) int {
+	baseLabelNumbers := []int{
+		// main menu labels
+		1612, // CANCEL
+		1613, // (c) 2000 Blizzard Entertainment
+		1614, // All Rights Reserved.
+		1620, // SINGLE PLAYER
+		1621, // BATTLE.NET
+		1623, // OTHER MULTIPLAYER
+		1625, // EXIT DIABLO II
+		1627, // CREDITS
+		1639, // CINEMATICS
+
+		// cinematics menu labels
+		1640, // View All Earned Cinematics
+		1659, // Epilogue
+		1660, // SELECT CINEMATICS
+
+		// multiplayer labels
+		1663, // OPEN BATTLE.NET
+		1666, // TCP/IP GAME
+		1667, // TCP/IP Options
+		1675, // HOST GAME
+		1676, // JOIN GAME
+		1678, // Enter Host IP Address to Join Game
+		1680, // Your IP Address is:
+		1689, // Tip: host game
+		1690, // Tip: join game
+		1691, // Cannot detect a valid TCP/IP address.
+		1694, // Character Name
+		1696, // Hardcore
+		1697, // Select Hero Class
+
+		1698, // amazon description
+		1704, // nec description
+		1709, // barb description
+		1710, // sorc description
+		1711, // pal description
+
+		/* these items are for battle.net multiplayer
+		labels and are not used yet
+		in addition, as many elements as the value
+		of the highest modifier must be replaced*/
+		1712, // not used here
```

Comment:
Why put this here, then? is this value used?

Reply:
sorry, wrong comment her